xml/ha_storage_protection.xml

```diff
@@ -698,6 +698,66 @@ Timeout (msgwait)  : 10
    </procedure>
   </sect1>
 
+  <sect1 xml:id="sec.ha.storage.protect.diskless-sbd">
+   <title>Setting up Diskless SBD</title>
+   <para>
+    SBD can be operated in a diskless mode. As diskless SBD is not able
+    to handle split-brain scenarios in a two-node cluster, it is required
+    for this fencing mechanism to use a minimum of three nodes.
+   </para>
+   <para>
+    When setting up SBD, you need to take several timeout values into account.
+    For details, see <xref linkend="sec.ha.storage.protect.watchdog.timings"/>.
+   </para>
+   <para>
+    The following steps are necessary to set up diskless storage-based
+    protection:
+   </para>
+
+   <procedure xml:id="pro.ha.storage.protect.confdiskless">
+    <title>Configuring Diskless SBD</title>
+    <step>
+     <para>Open the file <filename>/etc/sysconfig/sbd</filename> and use
+      the following entries:</para>
+     <screen>SBD_PACEMAKER=yes
+SBD_STARTMODE=always
+SBD_DELAY_START=no
+SBD_WATCHDOG_DEV=/dev/watchdog
+SBD_WATCHDOG_TIMEOUT=5</screen>
+      <para>Keep in mind, we do not use a shared disk. As such, we do not
```

Comment:
I would move this to a separate step and rephrase to: 
Comment the SBD_DEVICE entry. As no shared disk is used, we do not need it.